Gruntfile.js

```diff
@@ -216,6 +216,23 @@ module.exports = function (grunt) {
                     exclude: ["text!config.json"],
                     uglify2: {} // https://github.com/mishoo/UglifyJS2
                 }
+            },
+            iframe: {
+                // Standalone dist/bramble.js iframe api
+                options: {
+                    name: 'bramble/almond',
```

Comment:
Almond is a slimmed down shim for requirejs that you can use in deployment when you just want to use requirejs modules to make a standalone thing.  It's mostly about space-saving in production files.